app/views/entitlements/_entitlements_table.html.erb

```diff
@@ -18,20 +18,22 @@
 
 <table class="entitlements-table">
   <tbody>
-    <tr>
-      <td>
-        <h3>Total Weight of Household Goods</h3>
+    <%- if @entitlement_search.nil? or @entitlement_search.dependency_status == 'false' or !entitlement.total_weight_self_plus_dependents? -%>
```

Comment:
The conditionals in this file are getting a hairy. Is it possible to refactor these into one or more helpers and/or simplify the logic?